lib/src/message/message_item.dart

```diff
@@ -257,4 +293,13 @@ class _ChatMessageItemState extends State<ChatMessageItem> with AutomaticKeepAli
       ),
     );
   }
+
+  void _showErrorDialog(String messageInfo) {
+    showInformationDialog(
+      context: context,
+      title: L10n.get(L.error),
+      content: messageInfo,
+      navigatable: Navigatable(Type.messageInfoDialog),
+    );
+  }
```

Comment:
Naive question: Am I seeing it right that this function is only for `message_item`? Wouldn't it be better to have such generic functions to have in a global helper class (or whatever is being used the Dart-way-to-go), would it? // @Boehrsi

Reply:
The showInformationDialog is the mentioned helper we already wrote :), here we just use it with some defined strings to provide the error dialog. The helper method is quite flexible to be able to show errors, information and other stuff.